openshift/Makefile

```diff
@@ -14,9 +14,11 @@ REQUIREMENTS_IMG ?= pip-requirements
 
 .PHONY: build-requirements
 build-requirements:
-	cp ../images/ansible-operator/Pipfile* .
+	mkdir -p ./images/ansible-operator/
+	cp ../images/ansible-operator/Pipfile* ./images/ansible-operator/
 	$(CONTAINER_ENGINE) build -t $(REQUIREMENTS_IMG) -f Dockerfile.requirements .
-	rm Pipfile*
+	rm ./images/ansible-operator/Pipfile*
+	rm -R ./images/
```

Comment:
I think `rm -R ./images/` should be enough to remove the directory, do we really need two steps?